omod/src/main/webapp/resources/scripts/registerPatient.js

```diff
@@ -297,8 +297,25 @@ jq(function() {
     /* Unknown patient functionality */
     jq('#checkbox-unknown-patient').click(function () {
         if (jq('#checkbox-unknown-patient').is(':checked')) {
-            // disable all questions & sections except gender and registration date
-            if (NavigatorController.getQuestionById('demographics-name') != undefined) {    
+
+            $("#personRelationshipField h3").addClass("disabled");
+
+            $("#personRelationshipField p select").addClass("disabled");
+            $("#personRelationshipField p select").prop( "disabled", true);
+            $("#personRelationshipField p select").val(null);
+
+            $("#personRelationshipField p input").addClass( "disabled");
+            $("#personRelationshipField p input").prop( "disabled", true);
+            $("#personRelationshipField p input").val(null);
+
+            $("#personRelationshipField p a").each(function(i, button) {
+                if ((i % 2) != 0){
+                    button.click();
+                }
+            });

```

Comment:
@icrc-toliveira this clicks on the second button each time right, the "minus" one?

If so why not do that first? As in:
1. Click on all minus buttons, then
2. Disable and clear the last remaining relationship row.

Reply:
Ok, 
Done